packages/mongodb-chatbot-ui/src/ChatbotView.tsx

```diff
@@ -1,7 +1,9 @@
 import { DarkModeProps } from "./DarkMode";
 
 export type ChatbotViewProps = DarkModeProps & {
+  disclaimer?: React.ReactNode;
```

Comment:
nit: instead of `disclaimer` maybe we could give a more general name like `bottomContent` since it's not nec a disclaimer being put here

Reply:
This is specifically the disclaimer at the top of chat, not underneath the input bar. I can change to `termsOfUse` though.

<img width="1002" alt="Screenshot 2024-01-10 at 3 29 27 PM" src="https://github.com/mongodb/chatbot/assets/15657698/af55cd63-0c23-4b3e-a4ff-726b6553de3e">

Reply:
Added

- `InputBarTrigger.bottomContent` (for the disclaimer)
- `ModalView.inputBottomText` (for "this is an experimental chatbot" etc)
- `ModalView.disclaimerHeading` (for "Terms of Use" etc)